From ce6652fffc3fa97a33f5e8e8ac36c72036b0d50f Mon Sep 17 00:00:00 2001
From: Daniel Kolesa <daniel@octaforge.org>
Date: Sun, 14 Nov 2021 03:19:36 +0100
Subject: [PATCH] disable atomic modesetting by default

atomic modesetting still causes issues on musl; disable it by
default (it can still be forced with WESTON_FORCE_ATOMIC)
---
 libweston/backend-drm/kms.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libweston/backend-drm/kms.c b/libweston/backend-drm/kms.c
index c91e381..556ec58 100644
--- a/libweston/backend-drm/kms.c
+++ b/libweston/backend-drm/kms.c
@@ -1468,7 +1468,7 @@ init_kms_caps(struct drm_backend *b)
 		b->universal_planes = (ret == 0);
 	}
 
-	if (b->universal_planes && !getenv("WESTON_DISABLE_ATOMIC")) {
+	if (b->universal_planes && getenv("WESTON_FORCE_ATOMIC")) {
 		ret = drmGetCap(b->drm.fd, DRM_CAP_CRTC_IN_VBLANK_EVENT, &cap);
 		if (ret != 0)
 			cap = 0;
-- 
2.33.1

